### PR TITLE
Replace custom Multi Pane class with JavaFX TextFlow

### PR DESCRIPTION
**Pull request by rodvar**
- Refactor requested in https://github.com/bisq-network/bisq2/issues/1358
 - Replaced the custom multi label pane with TextFlow and an utilitary class to reuse the setup
 - apply changes on both usages
 - Allow trades to go through without checking reputation in DevMode

### CODE REVIEW

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/take_offer/review/TakeOfferReviewView.java [hunk 1]
```diff
@@ -189,6 +189,10 @@ protected void onViewAttached() {
         takeOfferStatusPin = EasyBind.subscribe(model.getTakeOfferStatus(), this::showTakeOfferStatusFeedback);
     }
 
+    private void updatePriceTextFlow(String priceText) {
```
Comment (djing-chan) [inline apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/take_offer/review/TakeOfferReviewView.java]: Why extract this to a method (single use)?

File: apps/desktop/desktop/src/main/java/bisq/desktop/util/TextFlowUtils.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package bisq.desktop.util;
```
Comment (djing-chan) [inline apps/desktop/desktop/src/main/java/bisq/desktop/util/TextFlowUtils.java]: I think the`bisq.desktop.components.controls` package would be a better place. Maybe a util subpackage there. We have also `bisq.desktop.common.utils` but that is more for not UI controls specific stuff.

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/take_offer/review/TakeOfferReviewView.java [hunk 3]
```diff
@@ -105,47 +105,47 @@ class TakeOfferReviewView extends View<StackPane, TakeOfferReviewModel, TakeOffe
 
         rowIndex++;
         Label priceDescription = new Label(Res.get("bisqEasy.takeOffer.review.price.price"));
-        priceDescription.getStyleClass().add(descriptionStyle);
+        priceDescription.getStyleClass().add(DESCRIPTION_STYLE);
         content.add(priceDescription, 0, rowIndex);
 
-        price = new MultiStyleLabelPane();
-        price.getStyleClass().add(valueStyle);
+        price = new TextFlow();
```
Comment (djing-chan) [inline apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/take_offer/review/TakeOfferReviewView.java:111]: Actually I don't see a reason why TextFlow is needed here. We have a rather static layout without expected linebreaks.

File: apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/trade_wizard/review/TradeWizardReviewView.java [hunk 4]
```diff
@@ -124,7 +125,7 @@ class TradeWizardReviewView extends View<StackPane, TradeWizardReviewModel, Trad
         priceDescription.getStyleClass().add(descriptionStyle);
         gridPane.add(priceDescription, 0, rowIndex);
 
-        price = new MultiStyleLabelPane();
+        price = new TextFlow();
```
Comment (djing-chan) [inline apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/trade_wizard/review/TradeWizardReviewView.java:128]: Actually I don't see a reason why TextFlow is needed here. We have a rather static layout without expected linebreaks.

**Review by djing-chan — CHANGES_REQUESTED** [review]

**Review by djing-chan — APPROVED** [review]
ACK, beside small nits

File: apps/desktop/desktop/src/main/java/bisq/desktop/util/TextFlowUtils.java (hunk 2 above)
Comment (rodvar) [reply apps/desktop/desktop/src/main/java/bisq/desktop/util/TextFlowUtils.java]: I agree, I couldn't find a better place when I was doing it I'll refactor it now

**Comment by rodvar** [comment]
applied review suggestions

**Review by djing-chan — APPROVED** [review]
utACK

**Pull request merged**